README.md

```diff
@@ -207,8 +209,27 @@ Example for sending data to EU site:
     datadog_api_key: "123456"
 ```
 
+APM
+---
+
+To enable APM with agent V5:
+
+```yaml
+datadog_config:
+    apm_enabled: "true" # has to be a string
+```
+
+To enable APM with agent V6:
```

Comment:
nit: let's put `Agent 6` before `Agent 5`, as is done for the process agent below :)